### PR TITLE
In memory test

### PR DESCRIPTION
**Pull request by JuliaKlompmaker**

### CODE REVIEW

**Review by baggekjr — APPROVED** [review]
This looks very nice babes

**Review by SarahVestergren — APPROVED** [review]
Very good job honey boo boo :)

File: src/Chirp.Infrastructure/CheepRepository.cs [hunk 1]
```diff
@@ -66,25 +50,12 @@ public void CreateAuthor(AuthorDTO author)
...
-        {
-            Name = author.Name,
-            Email = author.Email,
-            Cheeps = new List<Cheep>()
-        };
 
         var CheepModel = new Cheep
         {
-            Author = AuthorModel,
+            Author = await _context.Authors.FirstOrDefaultAsync(c=>c.Email == author.Email),
```
Comment (ClaraSoenderborg) [inline src/Chirp.Infrastructure/CheepRepository.cs:58]: we should probably look at this warning later :)

**Review by ClaraSoenderborg — APPROVED** [review]

**Review by AnnikaJensby — APPROVED** [review]
stunning <3

**Pull request merged**